scripts/patches/fb2k_utils.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/submodules/fb2k_utils/src/qwr/qwr_exception.h b/submodules/fb2k_utils/src/qwr/qwr_exception.h
+index 7e75b04..de7a01c 100644
+--- a/submodules/fb2k_utils/src/qwr/qwr_exception.h
++++ b/submodules/fb2k_utils/src/qwr/qwr_exception.h
+@@ -12,7 +12,7 @@ class QwrException
+ public:
+     template <typename... Args>
+     explicit QwrException( qwr::u8string_view errorMessage, Args&&... errorMessageFmtArgs )
+-        : std::runtime_error( fmt::format( errorMessage, std::forward<Args>( errorMessageFmtArgs )... ) )
```

Comment:
fmt change should not be needed, since this 'adapter' is needed only for constexpr contexts